client/post-editor/editor-html-toolbar/index.jsx

```diff
@@ -74,6 +75,13 @@ export class EditorHtmlToolbar extends Component {
 		document.addEventListener( 'click', this.clickOutsideInsertContentMenu );
 
 		this.toggleToolbarScrollableOnResize();
+
+		this.isIE11Detected = every( [
+			window,
+			document,
+			window.MSInputMethodContext,
+			document.documentMode,
+		] );
```

Comment:
two notes here:

 - I may have forgotten to suggest it, but I believe that there's no suitable reason to bring this in as a class property. the browser won't change during a session and so we can probably benefit from bringing it out into the module scope outside of the React component (a module-level "global")

 - this will fail if we're not in a browser and throw a `TypeError: MSInputMethodContext is not a property of undefined`. As much as I hate it we might want to go back to a short-circuited list…

```js
const isIE11Detected = (
	window &&
	window.MSInputMethodContext &&
	document &&
	document.documentMode
);
```